classes/helper/HelperList.php

```diff
@@ -623,7 +623,7 @@ public function displayListHeader()
                     if (is_string($value)) {
                         $value = json_decode($value, true);
                     }
-                    if (!Validate::isCleanHtml($value[0]) || !Validate::isCleanHtml($value[1])) {
+                    if (isset($value[0]) && !Validate::isCleanHtml($value[0]) || isset($value[0]) && !Validate::isCleanHtml($value[1])) {
```

Comment:
Wong index for second value `isset($value[0])`

Reply:
Yes!

> 
> 
> Wong index for second value `isset($value[0])`

Yes! Thx!